stdlib/public/Glibc/Glibc.swift

```diff
@@ -29,6 +29,82 @@ public var errno: Int32 {
 // fcntl.h
 //===----------------------------------------------------------------------===//
 
+@warn_unused_result
+@_silgen_name("_swift_Glibc_open")
+func _swift_Glibc_open(path: UnsafePointer<CChar>,
+  _ oflag: CInt, _ mode: mode_t) -> CInt
+
+@warn_unused_result
+@_silgen_name("_swift_Glibc_openat")
+func _swift_Glibc_openat(fd: CInt,
+  _ path: UnsafePointer<CChar>,
+  _ oflag: CInt, _ mode: mode_t) -> CInt
+
+@warn_unused_result
+public func open(path: UnsafePointer<CChar>, _ oflag: CInt) -> CInt {
+  return _swift_Glibc_open(path, oflag, 0)
+}
+
+@warn_unused_result
+public func open(path: UnsafePointer<CChar>, _ oflag: CInt,
```

Comment:
In Swift code, if you are breaking the signature, please put each parameter on a separate line:

```
public func open(
  path: UnsafePointer<CChar>,
  _ oflag: CInt,
  ...
) -> CInt {
  ...
}
```

(here and in other places)